exasol_advanced_analytics_framework/query_handler/context/proxy/bucketfs_location_proxy.py

```diff
@@ -16,5 +20,20 @@ def bucketfs_location(self) -> AbstractBucketFSLocation:
     def cleanup(self):
         if self._is_valid:
             raise Exception("Cleanup of BucketFSLocationProxy only allowed after release.")
-        for file in self._bucketfs_location.list_files_in_bucketfs(""):
-            self._bucketfs_location.delete_file_in_bucketfs(file)
\ No newline at end of file
+        files = self._list_files()
+        for file in files:
+            self._remove_file(file)
+
+    def _remove_file(self, file):
+        try:
+            self._bucketfs_location.delete_file_in_bucketfs(file)
+        except Exception as e:
+            LOGGER.error(f"Failed to remove {file}, got exception", exc_info=True)
+
+    def _list_files(self):
+        files = []
+        try:
+            files = self._bucketfs_location.list_files_in_bucketfs("")
+        except Exception as e:
+            LOGGER.debug(f"Got exception during listing files in temporary BucketFSLocation", exc_info=True)
```

Comment:
why only debug? Shouldn't be at least a warning?

Reply:
ah,  I catch Exception, I wanted to catch FileNotFound and make a debug and for Exception an error